app/api/helpers/export_helpers.py

```diff
@@ -241,12 +241,15 @@ def create_export_job(task_id, event_id):
     if export_job:
 
         export_job.task = task_url
-        export_job.user_email = current_user.email
+        if current_identity:
+            export_job.user_email = current_identity.email
+        else:
+            export_job.user_email = current_user.email
```

Comment:
Why this switching?

Reply:
To handle the test case. In the test case, the user is being logged in with the help of flask_login and doesn't use flask_jwt, hence the `current_identity` will be None in that case

Reply:
Create a common method which returns either of them so we can just call `get_current_user().email`

Reply:
@iamareebjamal Sure, on it.